software/github/BUILD

```diff
@@ -21,42 +21,42 @@
 
 java_binary(
     name = "github-bin-mac",
-    classpath_resources = ["//github/config/logback:logback-xml"],
+    classpath_resources = ["//software/github/config/logback:logback-xml"],
     data = [
-        "//github/datasets:github-data-json",
-        "//github/schemas:github-data-schemas",
+        "//software/github/datasets:github-data-json",
+        "//software/github/schemas:github-data-schemas",
     ],
-    main_class = "github.view.GitApplication",
+    main_class = "com.vaticle.typedb.example.software.github.view.GitApplication",
```

Comment:
Besides restructure, also conforming the package name to the established pattern.